app/config/parameters.yml.dist

```diff
@@ -2,10 +2,10 @@ parameters:
     database_driver:   pdo_mysql
     database_host:     127.0.0.1
     database_port:     ~
-    database_name:     symfony
+    database_name:     mental
     database_name_testing:     symfony
-    database_user:     root
-    database_password: ~
+    database_user:     mental
+    database_password: mental
```

Comment:
this breaks all ci tests (travis + scrutinizer), please revert or adjust travis/scrutinizer to use a different file as template

Reply:
I added a special test config and changed it in travis config